site/content/docs/contributing/getting-started.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Getting Started"
+menu:
+  main:
+    parent: "contributing"
+    identifier: "getting started"
+    weight: 1
+---
+# Getting Started
+
+Welcome! This guide covers how to get started contributing to kind.
+
+## 1. Read the Kubernetes community guidelines
+
+Make sure to read you read the [Kubernetes community guidelines][community].
+In specific, read through the [Kubernetes contributor guidelines][contributor].
+
+Additionally, note that kind is developed on [GitHub][github] and will require
+an account to contribute.
+
+## 2. Install Tools
+
+### Install Documentation Tools
+
+If you wish to contribute to the documentation, it is recommended but not 
+required to install [hugo], which we use to develop this site.
+
+Please see: https://gohugo.io/getting-started/installing/
+
+### Install Developer Tools
+
+If you wish to contribute to kind's code you will need to install the following:
+
+* `git`
+* `go`
+* `docker`
+
+#### Install git
+Install `git` on your local machine.
+You can check if `git` is already on your system and properly installed with 
+the following command:
+
+```
+git --version
+```
+This documentation is written using `git` version 2.17.1. 
+Your version may be different depending on your OS.
+
+#### Install or upgrade Go
+Install or upgrade [Go using the instructions for your operating system][golang].
+You can check if Go is in your system with the following command:
+
+```
+go version
+```
+This documentation is written using Go version 1.11+.
+
+#### Install or upgrade Docker
+If you haven't already, install the 
+[Docker software using the instructions for your operating system][docker].
+If you have an existing installation, check your version and make sure you have
+the latest Docker.
```

Comment:
[1]

Reply:
on the host the latest docker should be fine I think?

Reply:
it's fine.